test/agentchat/contrib/test_lmm.py

```diff
@@ -4,15 +4,14 @@
 from unittest.mock import MagicMock
 
 import pytest
+from conftest import MOCK_OPEN_AI_API_KEY
 
 import autogen
 from autogen.agentchat.conversable_agent import ConversableAgent
 
-from conftest import MOCK_OPEN_AI_API_KEY
-
 try:
-    from autogen.agentchat.contrib.img_utils import get_pil_image
     from autogen.agentchat.contrib.multimodal_conversable_agent import MultimodalConversableAgent
```

Comment:
Should we test what happens when LLMs with different modalities communicate with each other? e.g. `gpt-3.5` <-> `gpt-4-vision`

Reply:
Updated the `test_group_chat_with_lmm` and use a regular ConversableAgent there.